acados/ocp_nlp/ocp_nlp_sqp.c

```diff
@@ -1348,10 +1314,6 @@ int ocp_nlp_sqp(void *config_, void *dims_, void *nlp_in_, void *nlp_out_,
 #if defined(ACADOS_WITH_OPENMP)
     #pragma omp for
 #endif
-    for (ii = 0; ii < N; ii++)
-    {
-        config->dynamics[ii]->model_set(config->dynamics[ii], dims->dynamics[ii], nlp_in->dynamics[ii], "T", nlp_in->Ts+ii);
```

Comment:
Why did you remove this?

Reply:
because of incompatibility with `irk_gnsf`.
I put it back and added a note since it might brake the templating interface